backend/drm/drm.c

```diff
@@ -473,16 +473,19 @@ static bool drm_connector_test(struct wlr_output *output) {
 		}
 	}
 
-	if ((output->pending.committed & WLR_OUTPUT_STATE_BUFFER) && !conn->backend->parent) {
+	if (conn->backend->parent) {
+		// If we're running as a secondary GPU, we can't perform an atomic
+		// commit without blitting a buffer.
```

Comment:
Hmm, so test commits aren't possible in that configuration?

Reply:
We need the buffer which will be used for scan-out to do a test-only commit. So yeah, it's tricky to get one without doing a buffer copy. I'd rather disable any fancy features on secondary GPUs for now (which we don't have any right now, but in the future will be output layers, scaling/cropping, etc).